soc/wch/ch32v00x/soc.yml

```diff
@@ -0,0 +1,6 @@
+family:
+  - name: ch32v00x
+    series:
+      - name: ch32v00x
+        socs:
+          - name: ch32v003
```

Comment:
get rid of family and just have series and socs

Reply:
Fixed now - thanks.